examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,592 @@
+{
```

Comment:
<p>no no no. The purpose of the constraint function is to model constraints and return whether those are still true. But as it is regularly executed we can use it to update the plotter. Please clarify</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>